platform/node/src/node_request.cpp

```diff
@@ -52,7 +58,9 @@ void NodeRequest::HandleCallback(const Nan::FunctionCallbackInfo<v8::Value>& inf
 
     // Move out of the object so callback() can only be fired once.
```

Comment:
Actually I think it will make the refcounter negative on double invocation.

Reply:
I am not sure what you mean by this?

Reply:
if we call the callback twice by mistake on the Javascript side...